app/views/admin/attachments/edit.html.erb

```diff
@@ -9,7 +9,7 @@
 
   <% if attachment.html? %>
     <aside class="govuk-grid-column-one-third">
-      <%= simple_formatting_sidebar hide_inline_attachments_help: true, show_footnote_help: true, show_stat_headline_help: true %>
+      <%= simple_formatting_sidebar hide_inline_attachments_help: true, show_footnote_help: true, show_stat_headline_help: true, preview_design_system: preview_design_system?(next_release: true) %>
```

Comment:
Is `preview_design_system?(next_release: true)` actually needed here?

This entire file should be scoped to Design System stuff since it's `edit.html.erb` rather than `edit_legacy.html.erb`, so I think it should be okay to simply hardcode it?

```suggestion
      <%= simple_formatting_sidebar hide_inline_attachments_help: true, show_footnote_help: true, show_stat_headline_help: true, preview_design_system: true %>
```

Reply:
yea good point!